api/dp-openapi.yaml

```
@@ -678,6 +678,118 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /projects/{projectid}/checkout:
+    put:
+      tags:
+      - project
+      description: Check out a page for proofreading and return data
+      parameters:
+      - $ref: '#/components/parameters/projectid'
+      - $ref: '#/components/parameters/projectstate'
+      responses:
+        200:
+          description: page data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/proof_page_data'
+        '400':
+          $ref: '#/components/responses/InvalidValue'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /projects/{projectid}/validatetext:
+    put:
+      tags:
+      - project
+      description: Checks the given text for bad characters, returns data indicating them
```

```suggestion
      description: Checks the given text for invalid characters, returns data indicating them
```
I thought this was WordCheck, not the unicode checker at first, `invalid` might be slightly clearer?

---

made some changes to clarify this